filters/include/pcl/filters/approximate_voxel_grid.h

```diff
@@ -109,9 +109,9 @@ namespace pcl
     private:
       struct he
       {
-        he () : ix (), iy (), iz (), count (0) {}
-        int ix, iy, iz;
-        int count;
+        he () = default;
+        int ix{}, iy{}, iz{};
```

Comment:
when just using empty braces, its initialized with 0? I think I would prefer explicit showing it with a 0 in it, like with count below?